config/_ci/release/bump-package-json

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
+const path = require('path')
+const args = process.argv
+
+const { log, generatePackageJson } = require('./_utils')
+const packageJson = require(path.resolve(__dirname, '../../package.json'))
+const nextVersion = args[2]
+
```

Comment:
Can’t this file be avoided? I have no much experience with semantic release, but I expected it will bump version automatically